rpm/publish/publish.sh

```diff
@@ -42,10 +42,10 @@ EOF
 	# generate index
 	# locally
 	# disable this for now, as it's currently now used and generate errors
-	# createrepo --update -o "$RPM_WEBDIR" "$RPMDIR/"
+	# createrepo_c --update -o "$RPM_WEBDIR" "$RPMDIR/"
 	# on the server
 	# shellcheck disable=SC2029
-	ssh "${SSH_OPTS[@]}" "$PKGSERVER" createrepo --update -o "'$RPM_WEBDIR'" "'$RPMDIR/'"
+	ssh "${SSH_OPTS[@]}" "$PKGSERVER" createrepo_c --update -o "'$RPM_WEBDIR'" "'$RPMDIR/'"
```

Comment:
https://github.com/jenkins-infra/release/issues/202

Reply:
It does not seem that `$PKGSERVER` has `createrepo_c` installed, which is a prerequisite to the successful deployment of this change.

Reply:
> If no, could we use a Docker container for this instead?

It is just software. You can do anything you want, if you write the code.

In the short term, I suggest upgrading the box to Ubuntu 22.04.

Reply:
> In the short term, I suggest upgrading the box to Ubuntu 22.04.

This machine also hosts the Update Center: its not an easy task and would require unplanned effort for now.
Not that I would not want, but that would be a huge context switch for the team meaning we would have to plan it for next week.

Reply:
We can just switch back to createrepo and symlink it in the container, it’s API compatible as far as I understand

Reply:
make sense. Many thanks @timja @basil for the help and quick reactions!

Upgrading our boxes from 18.04 to 22.04 is a topic that I would like to see happen quite fast though!

Reply:
Err, I don't think createrepo is even needed in the docker container?

It is only executed over SSH?

Reply:
https://release.ci.jenkins.io/job/core/job/package/job/master/200/console AND https://github.com/jenkinsci/packaging/pull/318

Reply:
https://github.com/jenkins-infra/helpdesk/issues/2982 for info